controller/src/beerocks/master/tasks/association_handling_task.cpp

```diff
@@ -256,6 +253,45 @@ void association_handling_task::handle_response(std::string mac,
                                                 std::shared_ptr<beerocks_header> beerocks_header)
 {
     switch (beerocks_header->action_op()) {
+    case beerocks_message::ACTION_CONTROL_CLIENT_START_MONITORING_RESPONSE: {
+
+        auto response =
+            beerocks_header
+                ->addClass<beerocks_message::cACTION_CONTROL_CLIENT_START_MONITORING_RESPONSE>();
+        if (!response) {
+            TASK_LOG(ERROR)
+                << "addClass failed for cACTION_CONTROL_CLIENT_START_MONITORING_RESPONSE";
+            return;
+        }
+
+        if (!response->success()) {
+            ++attempts;
+            if (attempts >= max_attempts) {
+                TASK_LOG(ERROR) << "state START_RSSI_MONITORING reached maximum attempts="
+                                << attempts << " aborting task, and disconnecting client!";
+                son_actions::disconnect_client(database, cmdu_tx, sta_mac, original_parent_mac,
```

Comment:
That's quite a drastic move...
The problem is probably not with the station, but with the monitor (not ready yet?).
@tomereli, @arnout, what do you guys think?

Reply:
https://github.com/prplfoundation/prplMesh/pull/789#discussion_r376810230

Reply:
I'll remove the disconnect call.
https://github.com/prplfoundation/prplMesh/pull/789#discussion_r377014434